src/content/1.7/scale/webserver.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Optimize your webserver
+alias:
+- 1.7/optimizations/
+---
+
+# Optimize your webserver
+
+We often talk about optimizing the application side but less about the webserver.
+
+Which is a shame as there is a lot to gain from some simple configurations parameters. And you don't need to have hundred of visitors for them to be efficient, either to reduce your server's load or speed up your visitor's browsing.
+
+As we'll see, the configuration options we'll tweak are quite easy and should allow some nice improvements.
+
+# Webserver's function
+
+You may already be aware of, a webserver is a piece of software dedicated to satisfy world wide web requests.
+
+It usually works through the http (and https) protocol and delivers web pages to the clients (your web browser). 
+
+{{< figure src="../img/architecture.png" title="webserver's function" >}}
+
+An interesting point to mention is that, php being compiled server side, it often is the webserver's job to manage this task before sending html to the web browser.
+
+## Testing your optimizations
+
+There are plenty of ways to validate the gains of those improvements but a very simple one is to go check any page speed website.
```

Comment:
When I see this 😄I think "great ! let's try !" and I look for ... links to tools or services to do them. Could you add one or two speed check ?

Reply:
I did think about it but I did not want to influence anyone using any specific tool.

Reply:
Still, added a google search link, let me know what you think of it :p